### PR TITLE
Add lint configuration to readme

### PR DESCRIPTION
**Pull request by tobiasheine**
[PT-518](https://novoda.atlassian.net/browse/PT-518) 
### Description
Scope of this PR is to add lint support to the readme file.

### CODE REVIEW

File: docs/tools/android_lint.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Android Lint
+[Android Lint](https://developer.android.com/studio/write/lint.html) is a linter and static analysis tool for Android projects which can detect bugs
+and potential issues in code, resources and configuration files.
+
+Be aware that Lint just supports Kotlin since version 3.0.1 of the [Android Gradle Plugin](https://developer.android.com/studio/releases/gradle-plugin.html).
```
Comment (rock3r) [inline docs/tools/android_lint.md]: ❌ it's 3.1.0, not 3.0.1

I would rephrase for consistency with the other tools: `Please note that Android Lint only supports Kotlin projects from the command line since version 3.1.0 of the ...`

File: docs/tools/android_lint.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## Configure Android Lint
+Lint is configured through the `lintOptions` closure. It supports all [official](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.LintOptions.html)
+properties except `abortOnError`, `htmlReport` and `xmlReport`. These are hardcoded so lint won't break the build on its own and always generates reports.
```
Comment (rock3r) [inline docs/tools/android_lint.md]: Typo: `lint` -> `Lint`

File: docs/tools/android_lint.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Android Lint
+[Android Lint](https://developer.android.com/studio/write/lint.html) is a linter and static analysis tool for Android projects which can detect bugs
+and potential issues in code, resources and configuration files.
+
+Be aware that Lint just supports Kotlin since version 3.1.0 of the [Android Gradle Plugin](https://developer.android.com/studio/releases/gradle-plugin.html).
+
+## Configure Android Lint
+Lint is configured through the `lintOptions` closure. It supports all [official](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.LintOptions.html)
+properties except `abortOnError`, `htmlReport` and `xmlReport`. These are hardcoded so Lint won't break the build on its own and always generates reports.
```
Comment (tasomaniac) [inline docs/tools/android_lint.md]: `overriden` instead of `hardcoded` maybe?
`so that` instead of just `so`?
